nmostesting/suites/IS1101Test.py

```diff
@@ -303,585 +217,236 @@ def test_01_03(self, test):
+                                              headers={"Content-Type": "application/octet-stream"},
+                                              data=self.valid_edid)
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
 
-            for input_id in self.not_edid_connected_inputs:
-                valid, response = TestHelper.do_request(
-                    "GET", self.compat_url + "inputs/" + input_id + "/edid/base/"
-                )
-                if not valid:
-                    return test.FAIL("Unexpected response from the streamcompatibility API: {}".format(response))
-                if response.status_code != 204:
-                    return test.FAIL("The input {} edid base streamcompatibility request has failed: {}"
-                                     .format(input_id, response))
-            return test.PASS()
+            # Verify that /edid/base returns the last Base EDID put
+            valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if (
+                not valid
+                or response.status_code != 200
+                or response.headers["Content-Type"] != "application/octet-stream"
+            ):
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
+            if response.content != self.valid_edid:
+                return test.FAIL("The Base EDID of Input {}"
+                                 "doesn't match the Base EDID that has been put".format(inputId))
+
+            # Verify that /edid/effective returns the last Base EDID put
+            if self.get_effective_edid(test, inputId) != self.valid_edid:
+                return test.FAIL("The Effective EDID of Input {}"
+                                 "doesn't match the Base EDID that has been put".format(inputId))
+
+            # Delete the Base EDID
+            valid, response = self.do_request("DELETE", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
 
-        return test.UNCLEAR("No resources found to perform this test")
+            # Verify that the Base EDID is properly deleted
+            valid, response = self.do_request("GET", self.compat_url + "inputs/" + inputId + "/edid/base")
+            if not valid or response.status_code != 204:
+                return test.FAIL("Unexpected response from "
+                                 "the Stream Compatibility Management API: {}".format(response))
 
```

Comment:
Add reasonable delay after Delete command before getting the effective EDID
start_time = time.time()
while time.time() < start_time + CONFIG.API_PROCESSING_TIMEOUT:
time.sleep(0.2)